pages/accessibility.tsx

```diff
@@ -0,0 +1,54 @@
+import Head from 'next/head';
+
+export default function Accessibility() {
+  return (
+    <div className="gel-wrap accessibility-page gel-2/3">
```

Comment:
gel-wrap stops it going bigger than the biggest breakpoint width. 2/3 makes it always 2/3 of the width. Don't need both and 2/3 width on lower BPs looks a bit weird. 
Also applies to Privacy Policy page.